modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/constants/WorkflowWebKeys.java

```diff
@@ -22,6 +22,9 @@ public class WorkflowWebKeys {
 	public static final String SELECTED_WORKFLOW_PORTLET_TAB =
 		"SELECTED_WORKFLOW_PORTLET_TAB";
 
+	public static final String WORKFLOW_COMPARATOR_FACTORY =
```

Comment:
@JanisZhang Could you also pass the WorkflowComparatorFactory into the display context in this commit like in other commits? Try to see if you can move the creation of `[My]WorkflowInstanceEditDisplayContext` from jsp to Java class.